Bugzilla/User.pm

```diff
@@ -126,6 +126,10 @@ use constant EXTRA_REQUIRED_FIELDS => qw(is_enabled);
 
 with 'Bugzilla::Elastic::Role::Object';
 
+sub ES_INDEX {
+    Bugzilla->params->{elasticsearch_index} . "_user";
```

Comment:
`. "_" . ES_TYPE` would be a bit less repetitive of the word 'User', which is already available in the class name and then human-specified both here at +130 and below at +133 in ES_TYPE.